appendix/device-checklist.md

```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
 
 With each device, validate:
 
-- [ ] All computers have a complex password with a screen lock timeout set to no more than 10 minutes.
-- [ ] All computer hard drives are encrypted with FileVault, Bitlocker, or equivalent.
-- [ ] All operating systems and browsers should be patched and up to date.
-- [ ] Confirm VPN app (Proton VPN) is setup and the team member is aware of the "split tunnel" option.
-- [ ] Confirm 1Password is installed with browser extensions.
-- [ ] Confirm an adblocker (uBlock origin, AdGuard) is installed in the default browser.
+- [ ] Confirm all mobile devices have a [screen timeout set to no more than 10 minutes](https://security.lullabot.com/physical/locking.html).
+- [ ] Confirm "Find My" or equivalent is enabled.
+- [ ] All operating systems should be up to date.
+- [ ] All browsers should be up to date.
+- [ ] Confirm [1Password](https://security.lullabot.com/access/password_managers.html) is installed, with browser extensions.
+- [ ] Confirm an adblocker (uBlock origin lite, AdGuard) is installed in the default browser.
 
-## Workstations
+## Computers
 
-- [ ] Confirm "Find My" or equivalent is enabled.
-- [ ] Confirm firewall is enabled.
+- [ ] All computers have a [complex password](https://security.lullabot.com/access/passwords.html), at least 16 characters long. If you add your passwords to 1password, 1password will tell you the strenght of the password; try to get "excellent" strength. Do NOT use guessable patterns like: family names, birthdays, the word “lullabot”, etc.
+- [ ] All computer [hard drives are encrypted](https://security.lullabot.com/physical/hard-drive-encryption.html) with FileVault, Bitlocker, or equivalent.
+- [ ] Confirm firewall is enabled. Document work tasks that require disabling firewall; share with security@lullabot.com.
 - [ ] Confirm no guest accounts are enabled.
-- [ ] Confirm that if remote access is enabled (SSH, file sharing) that the implications are understood.
+- [ ] Confirm no remote access is enabled (SSH, file sharing). Document work tasks that require disabling firewall; share with security@lullabot.com.
+- [ ] Confirm automatic updates are enabled.
+- [ ] Confirm [VPN](https://lb.cm/vpn) app (Proton VPN) is setup.
 
 ## Backups
 
-- [ ] Computers have a backup system in place for quick recovery after a hardware failure.
-- [ ] Backups support versioning and have protections against ransomware style scams.
-- [ ] All backups are encrypted. Cloud services use a private key that only you know.
+- [ ] If computers have a [backup system](https://security.lullabot.com/physical/backups.html) in place, backups should be encrypted.
+- [ ] Optional, but nice to have: backups support versioning to protection against ransomware style scams.
+- [ ] If using cloud services, use a private key that only you know. (Store it in 1password.)
```

Comment:
```suggestion
- [ ] If using cloud services for backups, use a private key that only you know. (Store it in 1password.)
```

Reply:
Committed suggestion (manually, the button was not available and the hunk said "outdated".)